src/mainboard/clevo/adl-p/ramstage.c

```diff
@@ -147,6 +147,34 @@ static void set_camera_enablement(void)
 	system76_ec_smfi_cmd(CMD_CAMERA_ENABLEMENT_SET, sizeof(enabled) / sizeof(uint8_t), (uint8_t *)&enabled);
 }
 
+static bool get_wifi_bt_enable(void)
+{
+	bool enable = true;
+
+#if CONFIG(DRIVERS_EFI_VARIABLE_STORE)
+	struct region_device rdev;
+	enum cb_err ret;
+
+	uint32_t size = sizeof(enable);
+
+	const EFI_GUID dasharo_system_features_guid = {
+		0xd15b327e, 0xff2d, 0x4fc1, { 0xab, 0xf6, 0xc1, 0x2b, 0xd0, 0x8c, 0x13, 0x59 }
+	};
+
+	if (smmstore_lookup_region(&rdev))
+		goto efi_err;
+
+	ret = efi_fv_get_option(&rdev, &dasharo_system_features_guid, "EnableWifiBt",
+							&enable, &size);
```

Comment:
```suggestion
	ret = efi_fv_get_option(&rdev, &dasharo_system_features_guid, "EnableWifiBt",
				&enable, &size);
```

Please align all calls to efi_fv_get_option properly

Reply:
Aligned in d7f3d41ed9f54064d6c43eac17b77edd52f1a2cb